src/components/MovieCard/MovieCard.js

```diff
@@ -1,19 +1,24 @@
 import React from "react";
 import "./MovieCard.css";
 
-const MovieCard = ({id, title, poster, avgRating, releaseDate, handleClick}) => {
-    return (
-        <section className='movie-card'>
-            <div>
-                <span>{avgRating}</span>
-                <img src={poster} alt="movie poster"
-                onClick={event => {handleClick(event)}}
-                />
-            </div>
-            <h3>{title}</h3>
-            <span>{releaseDate}</span>
-        </section>
-    )
-}
+const MovieCard = ({ id, title, poster, avgRating, releaseDate, handleClick, submitId }) => {
+  return (
+    <section className="movie-card">
+      <div className="movie-card-poster-container">
+        <span className="avg-rating">🍅 <b>{avgRating}/10</b></span>
+        <img
+          src={poster}
+          alt="movie poster"
```

Comment:
Nice work including the alt tag for accessibility - can you use something more dynamic so its a more descriptive alt tag based on each movie?